terraform/ecr/main.tf

```diff
@@ -1,17 +1,17 @@
-# resource "aws_ecr_repository" "project_ecr" {
-#     name = "${var.projectname}-ecr"
-#     image_tag_mutability = "MUTABLE"
-#     image_scanning_configuration {
-#         scan_on_push = true
-#     }
-# }
-data "aws_ecr_repository" "ecr" {
```

Comment:
i'm pulling "data" here because of make prepare content..
you can't create ecr repo here if you already created on make prepare command.
i'll explain a bit further down :)